crypto-ffi/src/generic.rs

```diff
@@ -725,11 +723,9 @@ pub trait CoreCryptoCallbacks: std::fmt::Debug + Send + Sync {
 /// WARNING: This is a global setting. Calling it twice will cause errors.
 #[uniffi::export]
 pub fn set_logger(logger: std::sync::Arc<dyn CoreCryptoLogger>, level: CoreCryptoLogLevel) {
-    fmt::fmt()
-        .json()
-        .with_writer(CoreCryptoLoggerWrapper { logger, level })
-        .with_max_level(LevelFilter::from(level))
-        .init()
+    let logger = CoreCryptoLoggerWrapper { logger };
+    log::set_boxed_logger(Box::new(logger)).unwrap();
```

Comment:
I assume we shouldn't unwrap here, but I guess that means we need to start returning an error?

Reply:
Hm, yeah, good question.
Generally, it is frowned upon to just panic inside a library when something bad happens and instead one is encouraged to return an error so the caller can handle it.

On the other hand, in this case the error can only happen if a client calls this function twice, which is a programmer error and we anyway expect this to be replaced in the next iteration of our logging mechanism so I guess it's fine to leave it as-is.